sdk/core/Azure.Core/src/NullableResponseOfT.cs

```diff
@@ -23,7 +23,7 @@ public abstract class NullableResponse<T>
         /// <summary>
         /// Gets the value returned by the service. Accessing this property will throw if <see cref="HasValue"/> is false.
         /// </summary>
-        public abstract T Value { get; }
+        public abstract T? Value { get; }
```

Comment:
I suppose this is considered a low impact source breaking change?

Reply:
I don't think it is breaking since we already can return null here - this change makes the nullability accurate.

Reply:
It is a source breaking change, i.e. users will need to modify their source code and recompile.

Reply:
I think in some cases it could be breaking, but would mostly produce new warnings. These types of changes were [done by the BCL as well](https://learn.microsoft.com/en-us/dotnet/core/compatibility/core-libraries/6.0/nullable-ref-type-annotation-changes#change-description) so I think we are good with it.

Reply:
Agreed, but I was just confirming that this was an acceptable breaking change.

Reply:
@JoshLove-msft, for context, how widely would this affect people?  Is it anyone who accesses a `NullableResponse<T>.Value` property, so isolated to Tables customers?  Or is it broader than that?

Reply:
Yes, I think it would be restricted to Tables customers as the only library that currently exposes this type.